schema/examples/direct/manifest.xml

```diff
@@ -2,11 +2,14 @@
 <direct-signature-job-manifest xmlns="http://signering.posten.no/schema/v1">
     <signer>
         <personal-identification-number>12345678910</personal-identification-number>
+        <signing-ceremony>
```

Comment:
Legg merke til PR-tittelen («Forslag til …»), så det er fullstendig innafor å ta dette for vond fisk! ;)

Nei, bakgrunnen var vel at vi spekulerte i om det var flere attributer som kunne påvirke signeringssermonien, og at det kunne være hensiktsmessig å gruppere de. Meeeen, det er slettes ikke sikkert det er relevant så for min del kan vi godt legge det rett under `signer`.